templates/zerver/help/import-from-gitter.md

```diff
@@ -1,105 +1,126 @@
 # Import from Gitter (beta)
 
-Zulip supports importing data from Gitter, including users, channels,
-messages, attachments, and avatars.
+You can import your current workspace into a Zulip organization. It's a great way
+to preserve your workspace history when you migrate to Zulip, and to
+make the transition easy for the members of your organization.
 
-**Note:** You can only import a Gitter room as a new Zulip organization. In
-particular, this tool you cannot use this tool to import from Gitter into an
-existing Zulip organization.
+Zulip supports importing data from Gitter, including users, channels, messages,
+attachments, and avatars.
 
-## Import from Gitter
+## Import process overview
 
-First, export your data from Gitter.
+To import your Gitter organization into Zulip, you will need to take the
+following steps, which are described in more detail below:
+
+{start_tabs}
+
+1. [Export your Gitter data](#export-your-gitter-data).
+
+1. [Import your Gitter data into Zulip](#import-your-data-into-zulip).
+
+1. [Get your organization started with Zulip](#get-your-organization-started-with-zulip)!
+
+{end_tabs}
+
+## Import your organization from Gitter into Zulip
 
 ### Export your Gitter data
 
+Gitter's [data export tool](https://github.com/minrk/archive-gitter) allows you to
+export all public channel messages.
+
 {start_tabs}
 
-1. [Export your Gitter data](https://github.com/minrk/archive-gitter). You will
-   receive json files of the public rooms that you are a part of.
-   Select the `gitter_data.json` file of the room which you want to import into
-   Zulip.
+1. Export your Gitter data. You will receive json files of the public
```

Comment:
You will receive `.json` files...

That might be a bit better and follow e.g. the rocketchat doc convention of saying "`.bson` files"?